types/react-onsenui/index.d.ts

```diff
@@ -148,6 +150,38 @@ export class Navigator extends Component<{
     popPage(options?: PageTransitionOptions): Promise<HTMLElement>;
 }
 
+// Still incomplete, see https://onsen.io/v2/api/react/RouterNavigator.html
+export class RouterNavigator extends Component<{
+  routeConfig: any,
```

Comment:
I would naively have guessed that the type would be `RouteConfig` from below.

Reply:
Most probably - and according to the docs - yes. See my general comment in the PR thread.